languages/go/project.go

```diff
@@ -0,0 +1,118 @@
+package sdk
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type ProjectsInterface interface {
+	Create(organizationID string, name string) (ResponseForProjectResponse, error)
+	List(organizationID string) (ResponseForProjectsResponse, error)
+	Get(projectID string) (ResponseForProjectResponse, error)
+	Update(projectID string, organizationID string, name string) (ResponseForProjectResponse, error)
+	Delete(projectIDs []string) (ResponseForProjectsDeleteResponse, error)
+}
+
+type Projects struct {
+	CommandRunner CommandRunnerInterface
+}
+
+func NewProjects(commandRunner CommandRunnerInterface) *Projects {
+	return &Projects{CommandRunner: commandRunner}
+}
+
+func (p *Projects) Get(id string) (ResponseForProjectResponse, error) {
```

Comment:
Similar to the .net SDK, can we unwrap the responses and return the error in the library? It's somewhat annoying if consumers needs to handle two error types.